### PR TITLE
Set assignees on ART alignment PRs

### PR DESCRIPTION
**Pull request by jupierce**
Several improvements to ART alignment PRs.
- If no owner is assigned to the PR, assign the first approver in the OWNERS file.
- Keep the base branch up-to-date (this can change after branch cut)
- Close PR if it is no longer necessary

### CODE REVIEW

**Comment by jupierce** [comment]
@thegreyd thanks for bringing up the idea during the arch call!

**Review by vfreex — CHANGES_REQUESTED** [review]
generally LGTM, just 1 nit.

**Review by thiagoalessio — APPROVED** [review]
I also have the same question as Yuxiang about the duplicated line, but overall: /lgtm

File: doozerlib/cli/images_streams.py [hunk 1]
```diff
@@ -654,17 +662,25 @@ def images_streams_prs(runtime, github_access_token, bug, interstitial, ignore_c
                 # If there is already an art reconciliation branch, get an MD5
                 # of the FROM images in the Dockerfile in that branch.
                 exectools.cmd_assert(f'git checkout fork/{fork_branch_name}')
-                fork_branch_parent_digest, fork_branch_parents = extract_parent_digest(df_path)
                 fork_branch_df_digest = compute_dockerfile_digest(df_path)
 
             # Now change over to the target branch in the actual public repo
             exectools.cmd_assert(f'git checkout public/{public_branch}')
             source_branch_parent_digest, source_branch_parents = extract_parent_digest(df_path)
+            exectools.cmd_assert(f'git checkout public/{public_branch}')
```
[the remark this answers is not in the document]
Comment (jupierce) [reply doozerlib/cli/images_streams.py]: If it's worth doing once, why not twice?  :)
Thanks for the review! Removed the line.

**Pull request merged**